New log file opened at Wed May 06 11:05:24 CEST 2015 by 8363@c126-13
Taking next connection to: 51816
New client attempting to connect on port: 35784-51816
  Address = /137.194.35.173:51816
This client on port 35784 has uuid 149c3dca-7999-46d2-9ea8-d05e40e2bd25 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-149c3dca-7999-46d2-9ea8-d05e40e2bd25
Taking next connection to: 51816
New client attempting to connect on port: 35785-51816
  Address = /137.194.35.173:51816
Handshake exception on socket: 35785
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 51816
New client attempting to connect on port: 35786-51816
  Address = /137.194.35.173:51816
This client on port 35786 has uuid 620e85be-425e-4888-a6f3-e6edf84ba274 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-149c3dca-7999-46d2-9ea8-d05e40e2bd25, activator-620e85be-425e-4888-a6f3-e6edf84ba274
Taking next connection to: 51816
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 51816
